goto/gotomagic/text/error.py

```diff
@@ -0,0 +1,59 @@
+
```

Comment:
Great!  I wanted to create this file, but I tried to not do too many things at once. Glad you did it though 👍